app/assets/stylesheets/design_system_vibereport.scss

```diff
@@ -50,7 +50,7 @@ h1 {
 }
 
 .b3 {
-  font-size: 1.5rem !important; //24px
+  //font-size: 1.5rem !important; //24px
```

Comment:
It's better to delete than to comment out stuff. If a mistake is made we can always use git blame to figure out the change and revert it back.